test/test-lib/init_mocks.ts

```diff
@@ -1,7 +1,51 @@
-import './aws-mock';
+import { app } from '../../init';
+import * as balenaToken from './balena-token';
+import * as fixtures from './fixtures';
+import { supertest, UserObjectParam } from './supertest';
 
-// override the interval used to emit the queue stats event...
-import { DeviceOnlineStateManager } from '../../src/lib/device-online-state';
-(DeviceOnlineStateManager as any)['QUEUE_STATS_INTERVAL_MSEC'] = 1000;
+export const preInit = async () => {
+	await import('./aws-mock');
 
-export {};
+	// override the interval used to emit the queue stats event...
+	const { DeviceOnlineStateManager } = await import(
+		'../../src/lib/device-online-state'
+	);
+	(DeviceOnlineStateManager as any)['QUEUE_STATS_INTERVAL_MSEC'] = 1000;
+};
+
+const getAdminUser = async () => {
+	// any user we try to create will be the superuser...
+	const { SUPERUSER_EMAIL, SUPERUSER_PASSWORD } = await import(
+		'../../src/lib/config'
+	);
+
+	if (!SUPERUSER_EMAIL || !SUPERUSER_PASSWORD) {
+		return;
```

Comment:
Can you please change this so it crashes if these values aren't set? It's going to surprise people if the admin user doesn't exist when it's referenced in tests.